core/src/main/java/org/apache/druid/math/expr/Exprs.java

```diff
@@ -29,6 +29,9 @@
 
 public class Exprs
 {
+  public static final byte EXPR_CACHE_KEY = 0x00;
+  public static final byte LOOKUP_EXPR_KEY = 0x01;
```

Comment:
Naming? LOOKUP_EXPR_KEY should be something like LOOKUP_CACHE_KEY, I think. For symmetry.

Reply:
oops, meant to rename to `LOOKUP_EXPR_CACHE_KEY` when i moved here, fixed